### PR TITLE
fixed dead-links, formatting, and errors

### PR DESCRIPTION
**Pull request by nogira**

### CODE REVIEW

File: content/guides/core/hoon-school/B-syntax.md [hunk 1]
```diff
@@ -543,7 +543,7 @@ Carefully map how the runes in that statement relate to each other, and notice h
 
 ##  Exercise:  “Absolute” Value (Around Ten)
```
Comment (matildepark) [inline content/guides/core/hoon-school/B-syntax.md:545]: It’s with ^s.

Comment (nogira) [reply content/guides/core/hoon-school/B-syntax.md:545]: that one's for superscript but i need subscript

its ok tho, while `₁₀` looks a little janky in monospace font, it looks fine in normal font ₁₀

**Comment by nogira** [comment]
i think someone forgot to mention what the `lent` gate does before this exercise (?) https://developers.urbit.org/guides/core/hoon-school/G-trees#exercise-evaluating-expressions

the exercise is right after a section where several list operators are explained (but not `lent`)

File: content/guides/core/hoon-school/B-syntax.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ nodes = [110, 113]
 objectives = ["Distinguish nouns, cells, and atoms.", "Apply auras to transform an atom.", "Identify common Hoon molds, such as cells, lists, and tapes.", "Pin a face to the subject.", "Make a decision at a branch point.", "Distinguish loobean from boolean operations.", "Slam a gate (call a function)."]
 +++
 
-_This module will discuss the fundamental data concepts of Hoon and how programs effect control flow._
```
Comment (sigilante) [inline content/guides/core/hoon-school/B-syntax.md:8]: Intentional but obscure.

**Review by sigilante — APPROVED** [review]

**Pull request merged**